manifests/y/yaser01/MKVMuxingBatchGUI/2.3/yaser01.MKVMuxingBatchGUI.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: yaser01.MKVMuxingBatchGUI
+PackageVersion: "2.3"
+PackageLocale: en-US
+Publisher: yaser01

```

```suggestion
Publisher:  bebotech, Inc.
```
This should match control panel